test/gradcheck.jl

```diff
@@ -1219,6 +1219,18 @@ end
   @test gradient(x -> findall(ismissing, x)[1], [1, missing]) == (nothing,)
 end
 
+@testset "nograd" begin
+  let n = 5, A = randn(n), x = randn(n)
+    function f(x)
+      Zygote.nograd() do
+        A .= x
+      end
+      sum(A + x)
+    end
+    @test f'(x) == ones(n)
```

Comment:
let's use `gradient`  instead of `'` as we may deprecate it in the future

Reply:
Fixed